metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -258,6 +258,11 @@ class SQLAlchemyConfig(StatefulIngestionConfigBase):
         default=True, description="Whether tables should be ingested."
     )
 
+    include_table_location_lineage: bool = Field(
```

Comment:
Do any other sources need to have the capability listed?

Reply:
Athena is the only source that generates lineage in this way